lib/devcycle-ruby-server-sdk/localbucketing/dvc_options.rb

```diff
@@ -43,10 +45,13 @@ def initialize(event_flush_interval_ms = 1000,
         max_event_queue_size = 10000
       end
       @max_event_queue_size = max_event_queue_size
+
       if flush_event_queue_size <= 0
         flush_event_queue_size = 1000
       end
 
+      @logger = logger || defined?(Rails) ? Rails.logger : Logger.new(STDOUT)
```

Comment:
do we want to create another logger instance here? There's already one on the configuration property of the api_client.  I think we should only have one so maybe remove the other one?

Reply:
I'm not sure, apparently it's per file? Not sure the best way to do so.

Reply:
i don't think it's per file. We can have a single instance and pass it around. We can just handle this in another thickets since it's not really related. Everything else looks good!